.github/workflows/buildkit.yml

```diff
@@ -151,6 +151,8 @@ jobs:
           targets: integration-tests
           set: |
             *.output=type=docker,name=${{ env.TEST_IMAGE_ID }}
+        env:
+          GO_VERSION: "1.24.3"
```

Comment:
Is this needed if we already have the `GO_VERSION` declared at the top?

Reply:
🤔 trying to recall why I did this; perhaps this was related to the pre-releases (when I first created this branch, go1.24 was still a pre-release, which uses a different format for the versions?)